content/en/cloud_cost_management/google_cloud.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Google Cloud
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/aws"
+  tag: "Documentation"
+  text: "Gain insights into your AWS bill"
+- link: "/cloud_cost_management/azure"
+  tag: "Documentation"
+  text: "Gain insights into your Azure bill"
+---
+
+{{< beta-callout url="https://docs.google.com/forms/d/e/1FAIpQLScNSzkTAmmpmxEY5Xoo-xPOyKXG0n_6IgdlU66mWXp445pNdA/viewform" btn_hidden="false" >}}
+Cloud Cost Google Cloud support is in private beta
+{{< /beta-callout >}}
+
+## Overview
+
+To use Cloud Cost Management with Google Cloud, you must set up a Datadog Google Cloud Platform Integration in Datadog. You must also enable a Detailed Usage Cost billing export for your desired billing account in Google Cloud.
+
+The Google Cloud Platform Datadog Integration Service Account must have necessary permissions configured to interact with Google Cloud Storage and BigQuery.
+
+## Setup
+
+### Enable Detailed Usage cost billing export
+<div class="alert alert-info"> <a href="https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage">Detailed Usage Cost</a> data includes resource-level cost data. </div>
+
+1. Navigate to [Billing Export][1] under Google Cloud console *Billing*
+2. Enable [*Detailed Usage cost*][2] export, selecting or creating a project and a BigQuery dataset
+3. Document the billing account id for the billing account where the export was configured as well as export project id and dataset name
```

Comment:
```suggestion
3. Document the billing `account id` for the billing account where the export was configured, as well as the export `project id` and dataset name.
```

Reply:
![Screenshot 2023-10-20 at 1 43 45 PM](https://github.com/DataDog/documentation/assets/3358140/4c918523-be2e-4688-8975-208b1bfa2a9f)
![Screenshot 2023-10-20 at 1 44 27 PM](https://github.com/DataDog/documentation/assets/3358140/41c10e93-63f7-4d70-803b-ebd5f0ecd258)

Updating to `billing account id`, `project id` and `dataset name`